MatrixSDK/JSONModels/Event/Content/MXEventContentLocation.m

```diff
@@ -63,16 +72,12 @@ + (instancetype)modelFromJSON:(NSDictionary *)JSONDictionary
     }
     
     NSDictionary *assetDictionary = JSONDictionary[kMXMessageContentKeyExtensibleAssetMSC3488];
-    if (assetDictionary == nil)
-    {
-        assetDictionary = JSONDictionary[kMXMessageContentKeyExtensibleAsset];
-    }
```

Comment:
I'm probably missing context, but isn't this removing support for the stable event type if the dictionary contained `m.asset` instead of `org.matrix.msc3488.asset`?

Reply:
I don't think so, and according to MSC https://github.com/matrix-org/matrix-spec-proposals/blob/matthew/location/proposals/3488-location.md#proposal when m.asset is missing, it should be treated like "m.self" and it was not done